crowbar_framework/app/controllers/nodes_controller.rb

```diff
@@ -139,6 +139,11 @@ def bulk
               dirty = true
             end
 
+            unless node.default_fs == node_attributes["default_fs"]
```

Comment:
I don't believe that we want to provide it on the bulk edit section as we already have not enough space.